src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp

```diff
@@ -701,6 +706,7 @@ void CompiledKernelsCacheTest::SetUp() {
 void CompiledKernelsCacheTest::TearDown() {
     std::remove(cache_path.c_str());
     core->set_property(ov::cache_dir());
+    ov::test::utils::PluginCache::get().reset();
```

Comment:
May be to replace by `core` creation which are defined in https://github.com/openvinotoolkit/openvino/pull/22595 to avoid of cache reset?

Reply:
The problem, which described here https://jira.devtools.intel.com/browse/CVS-120240 , is that if tests run after caching_tests, cache_dir can applied to plugins [here](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/dev/core_impl.cpp#L1488), even if the value is nullptr, so we need to clean cache_dir properly and deleting the property is not enough. If I understand correctly, creating core by new way solve the problem for these tests, but not for random next.